src/controllers/items.py

```diff
@@ -282,13 +285,13 @@ class ResetResponse(BaseModel):
     description="Reset media items with bases on item IDs",
     operation_id="reset_items",
 )
-async def reset_items(request: Request, ids: str) -> ResetResponse:
+async def reset_items(request: Request, ids: int) -> ResetResponse:
```

Comment:
thats what the input is, so then it looks like frontend is wrong then lol. I'll revert but a frontender will need to look at it then. @davidemarcoli @filiptrplan

Reply:
It can be int but you need to regenerate the frontend client then. It doesn't really matter that much what the datatype is. I imagine it was string so we can reset multiple ids at once in the future. For something like a bulk edit.

Reply:
Please revert this change